examples/configs/ragas/ragas_config.yaml

```diff
@@ -0,0 +1,38 @@
+dataset:
+  load_from:
+    name: "wandb_file_artifact_name.json"
+    version: "latest"
+    project: "lm-buddy-prometheus"
+    entity: "mozilla-ai"
+  # field containing scoring instructions in the json file
+  text_field: "instruction"
+
+judge:
+  # vllm evaluation server with model name
+  inference:
+    base_url: "http://your.vllm.server:8000/v1"
+    engine: "DiscoResearch/mixtral-7b-8expert"
+  max_tokens: 4000
+  temperature: 0.7
+  top_k: 1
+
+evaluation:
+  #choose which metrics to report from https://docs.ragas.io/en/latest/concepts/metrics/index.html
+  metrics: [
+        "faithfulness",
+        "answer_relevancy",
+        "context_recall",
+        "context_precision",
+    ]
+  # dummy openAI api key, needed for ragas shenanigans
+  openai_api_key: "nokey"
```

Comment:
we can set to "EMPTY" following lm-harness

Reply:
Do we need this on the config? If running on Ray, a user can inject this into their workers by the `runtime_env` attribute.

We aren't passing other keys (W&B, HF) as part of the configs, we are using env vars.

Reply:
unfortunately it's an artifact of Ragas, you need a dummy key when hitting non openai endpoints

Reply:
Same for prometheus' openai client! I solved it by just adding the following to my `~/.zshrc` file:
```
export OPENAI_API_KEY="Ceci n'est pas une openai key"
```
... does that work in your case?

Reply:
Yes if its just a dummy key, can we either set it as an env var or just hard-code a dummy for the client? it seems inapprorpiate to pass the dummy via the config since its not _really_ a user-controlled parameter.

Reply:
I like the idea of hardcoding it in. If we want to support actually using OpenAI models later we can add that in

Reply:
Simply removing the key from the yaml file passes the default, so I'll just remove it from the config yaml.